sbin/cubeit

```diff
@@ -235,6 +238,24 @@ if [ "${#ROOTFS_LABEL}" -gt 16 ]; then
 	fi
 fi
 
+# Check if interactive mode will be used
+if [ -z ${CONFIG_FILES} ] || ([ -n "$INTERACTIVE_MODE" ] && [ "$INTERACTIVE_MODE" -eq 1 ]); then
```

Comment:
Why is this same check/trigger for interactive mode being run from both cubeit and cubeit-installer ?